compiler/compile/OMRSymbolReferenceTable.cpp

```diff
@@ -1303,7 +1303,7 @@ OMR::SymbolReferenceTable::findOrCreateClassSymbol(
    sym->setClassObject();
 
 #ifdef J9_PROJECT_SPECIFIC
-   if (cpIndex == -1 && comp()->compileRelocatableCode())
+   if (cpIndex == -1 && comp()->compileRelocatableCode()
```

Comment:
Now you've removed a `)`. Did you try and compile this? It shouldn't compile as-is.